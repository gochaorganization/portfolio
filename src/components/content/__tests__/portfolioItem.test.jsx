import React from 'react'
import { shallow, mount } from 'enzyme'
import PortfolioItem from '../portfolioItem'

describe('PortfolioItem tests', () => {
  const props = {
    portfolio: {
      id: 1,
      src: 'https://placekitten.com/380/380',
      title: 'Title',
      subtitle: 'subtitle',
    },
  }

  it('renders correctly', () => {
    const content = shallow(<PortfolioItem {...props} />)
    expect(content).toMatchSnapshot()
  })

  test('renders title', () => {
    const content = shallow(<PortfolioItem {...props} />)

    expect(content.find('.portfolio-item__title').text()).toEqual(
      props.portfolio.title
    )
  })
})
