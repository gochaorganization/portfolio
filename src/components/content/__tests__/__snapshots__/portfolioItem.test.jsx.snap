// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PortfolioItem tests renders correctly 1`] = `ShallowWrapper {}`;
